riak_test/yz_pb.erl

```diff
@@ -75,12 +80,27 @@ http(Method, URL, Headers, Body) ->
 create_index(Cluster, BucketType, Index) ->
     create_index(Cluster, BucketType, Index, 3).
 
-create_index(Cluster, BucketType, Index, Nval) ->
+create_index(Cluster, BucketType, Index, UseDefaultSchema) when
+      is_boolean(UseDefaultSchema) ->
+    create_index(Cluster, BucketType, Index, 3, UseDefaultSchema);
+create_index(Cluster, BucketType, Index, Nval) when is_integer(Nval) ->
+    create_index(Cluster, BucketType, Index, Nval, true).
+
+create_index(Cluster, BucketType, Index, Nval, UseDefaultSchema) ->
     Node = select_random(Cluster),
     [{Host, Port}] = host_entries(rt:connection_info([Node])),
     lager:info("create_index ~s for bucket type ~s [~p]", [Index, BucketType, {Host, Port}]),
     {ok, Pid} = riakc_pb_socket:start_link(Host, (Port-1)),
-    SchemaName = ?YZ_DEFAULT_SCHEMA_NAME,
+    if
+        UseDefaultSchema ->
+            lager:info("Using Default Schema"),
+            SchemaName = ?YZ_DEFAULT_SCHEMA_NAME;
+        true ->
+            lager:info("Using Custom Schema ?SCHEMA_CONTENT"),
+            SchemaName = <<"anotherschema">>,
```

Comment:
(or, alternatively, if you expect to write more tests requiring different custom schemas, parameterize the function on SchemaName/SchemaContent, perhaps in a 2-tuple, rather than a boolean for toggling default vs alternate)